api/src/main/java/io/opentelemetry/trace/Event.java

```diff
@@ -41,4 +41,11 @@
    * @since 0.1.0
    */
   Map<String, AttributeValue> getAttributes();
+
+  /**
+   * Returns the number of dropped attributes.
+   *
+   * @return the number of dropped attributes.
+   */
+  long getDroppedAttributeCount();
```

Comment:
An int should definitely be sufficient, I would think, to hold the number of dropped attributes.

Reply:
Yeah, not sure why I picked long. I'll update to int.